src/store.ts

```diff
@@ -137,22 +137,24 @@ export class DiscordStore {
 
   public get_user_discord_ids(userId: string): Promise<string[]> {
     log.silly("SQL", "get_user_discord_ids => %s", userId);
-    return this.db.getAsync(
+    return this.db.allAsync(
       `
       SELECT discord_id
       FROM user_id_discord_id
-      WHERE user_id = $userId
+      WHERE user_id = $userId;
       `, {
         $userId: userId,
       },
     ).then( (rows) => {
       if (rows !== undefined) {
-        rows.map((row) => row.discord_id);
+        const ret = [];
+        rows.forEach((row) => ret.push(row.discord_id));
+        return ret;
```

Comment:
Is there any reason why not to use map? I should be returning the result however -_-

Reply:
Not particularly, apart from "the bridge is broken and I know this will work, despite being stupid"...which is not really an acceptable standard for PRs, so I changed it.